_toc.yml

```diff
@@ -7,12 +7,20 @@ entries:
   - file: docs/tools/index
     entries:
       - file: docs/tools/cli
+        entries: 
+        - file: docs/tools/cli-fork-service.rst
```

Comment:
I suggest this could be a platform howto (I think this has changed since you started this PR)